test/unit/LivepeerETHTicketBroker.js

```diff
@@ -14,12 +15,13 @@ contract("LivepeerETHTicketBroker", accounts => {
     const recipient = accounts[1]
 
     const unlockPeriod = 20
+    const signerRevocationPeriod = 10
```

Comment:
Any reason for this value to be different than `unlockPeriod`? (Not a big deal, but if there isn't a reason then perhaps we just use the same constant values when possible to avoid future speculation on whether there is a reason for some value in tests or not)

Reply:
No special reason. I set a different value just because I thought to signal to readers that this param won't necessarily be the same duration as `unlockPeriod`, but simply the fact that you didn't like it when you read it is sufficient reason to change it to be the same :)